src/index.ts

```diff
@@ -26,6 +27,11 @@ export interface RawModule {
 const rootFields = ['Query', 'Mutation', 'Subscription']
 
 const read = (schema: string, schemas?: { [key: string]: string }) => {
+  if (schema.includes('*')) {
```

Comment:
What if the name of the actual schema file contains a `*`? Would that not be allowed?

Reply:
Good question, I don't have this in mind, I never had any `*` in my file name ^^

Reply:
Reason I'm asking is because we actually forked this repo and integrated many of the open PRs in here - including this one: https://www.npmjs.com/package/@flybondi/graphql-import

I know you've already published your own version, but when merging your changes  back into our own fork, we found out a couple of things you may want to have in mind:

1) Situation outlined above. Asking for `*` presence to spot a glob seems a bit of a stretch. AFAIK, there is no good solution to this, but we ended up changing the `includes` to a `match` and a stricter RegExp.

2) Your solution works well when all the root fields (`Query`, `Mutation`, `Subscription`, `schema`) _are all present in a single file_. If they are not (which is usually the case when splitting your schema), they don't get merge as opposed to how it'd work if you don't use globs. This breaks current functionality and should be used with care.

Reply:
Good to know! I split my schemas by "domains" so I don't run in this kind of issues 😅 

To be honnest I don't really have time anymore to update this PR, but if you have a fix for this merge problem, I will be happy to accept a PR in my PR 😄